### PR TITLE
July 27 morning: Android Agent v6.8.0 release notes

### PR DESCRIPTION
**Pull request by ywang-nr**
From @zuluecho9: I reviewed this, should be ready to publish. Doublecheck the release of agent is happening. 

## Give us some context

* What problems does this PR solve? - Android v6.8.0 release notes. [NR-31359]

### CODE REVIEW

**Comment by ywang-nr** [comment]
@cthomas-newrelic for review
@ndesai-newrelic for review

**Review by ndesai-newrelic — DISMISSED** [review]

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-671.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## New in this release
+* Add support for handling JavaScript Errors from React Native apps
+* Add harvest data usage supportibility metrics
+* Add new API to detect whether the hosting device is rooted
+
+
+## Fixed in this release
+* Fixed an issue with distributed tracing header
+* Fixed an issue to address plugin concurrency
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-671.mdx:16]: We didn't really. What that change did was address a corner-case exception thrown during map processing.

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-671.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+subject: Android agent
+releaseDate: '2022-07-27'
+version: 6.7.1
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.7.1.zip'
+---
+
+## New in this release
```
[the remark this answers is not in the document]
Comment (ywang-nr) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-671.mdx:8]: okay, will change the version number

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-671.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+* Add support for handling JavaScript Errors from React Native apps
+* Add harvest data usage supportibility metrics
```
[the remark this answers is not in the document]
Comment (ywang-nr) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-671.mdx:10]: I will take this out as John is out for the week

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-671.mdx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+* Add new API to detect whether the hosting device is rooted
```
[the remark this answers is not in the document]
Comment (ywang-nr) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-671.mdx]: I modified the language in the release notes

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-671.mdx (hunk 1 above)
Comment (ywang-nr) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-671.mdx]: I modified the language in the notes

**Comment by ywang-nr** [comment]
Release notes updated upon requests for review

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-680.mdx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+subject: Android agent
+releaseDate: '2022-07-27'
+version: 6.8.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.8.0.zip'
+---
+
+## New in this release
+* Add support for handling JavaScript Errors from React Native apps
+* Add a new session attribute 'RootedDevice' to signify whether the hosting device is rooted
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-680.mdx:10]: One last change: "Add a new session attribute _when native reporting is enabled_..."

**Review by cthomas-newrelic — DISMISSED** [review]
One minor change request, but ready to release!

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-680.mdx (hunk 5 above)
Comment (ywang-nr) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-680.mdx:10]: okay, will do

**Comment by ywang-nr** [comment]
@Doc Team, this release note is ready for doc team to review

**DO NOT MERGE YET**

**Review by zuluecho9 — APPROVED** [review]

**Pull request merged**